### ISSUE TITLE
[Accessibility]A11y_ Azure Data Studio_ NewNotebook_Keyboard:Keyboard focus is not visible after "Clear Results" button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5616]
[Check out Keros! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

**Additional Details:**
MAC Version: 10.14.5
Tool name: Keyboard
MAS Number: MAS2.4.7

**Steps to Reproduce:**
1. Install the app Azure Data Studio
2. Connect to the Database->Go to file->New Notebook
3. Navigate to the "Clear Results" button through tab.
4. Observe that after "Clear Results" button ,keyboard focus is not visible.

**Actual:**
Navigate till the "Clear Results" button through tab ,after clicking tab again the keyboard focus is not visible .When tab press again it is visible on “Code ” link .

**Expected:**
After clicking on tab the focus should move to the “Code ” link.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Keyboard only users find it difficult to navigate the page.

****MAS Reference:**
MAS2.4.7 :**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment for Reference:**
[5616_[Accessibility]A11y_ Azure Data Studio_ NewNotebook_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3215823/5616_.Accessibility.A11y_.Azure.Data.Studio_.NewNotebook_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by AccessibilityTestingTeam-TCS** [comment]
This issue is observed in "MAC 10.14.5". The issue is related to the loss of focus. When the keyboard focus is on "Clear Results" button then after clicking on tab key the focus is not visible on +Code button.
So, this is not the expected behaviour, the focus should be visible on each actionable control without any loss.

**Comment by chlafreniere** [comment]
@AccessibilityTestingTeam-TCS I have verified that, in a new notebook, focus goes directly from "Clear Results" to "+ Code", and does not disappear. Please ensure you have the latest insiders build.

**Comment by AccessibilityTestingTeam-TCS** [comment]
I checked this issue on following environment 
Version: 1.9.0-insider (user setup)
Commit: ad805b993c95e65e600921145c4f8d7602205ab7
Date: 2019-07-11T07:43:05.264Z
VS Code 1.34.0
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
and the issue is still reproduced.

**Comment by AccessibilityTestingTeam-TCS** [comment]
I checked this issue on following environment:
Version: 1.10.0-insider (user setup)
Commit: f720ec642f6ffd51d8772f663a611292f91a563d
Date: 2019-07-15T07:47:15.691Z
VS Code 1.34.0
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
and the issue is Fixed.
Hence closing the bug.
[5616_[Accessibility]A11y_ Azure Data Studio_ NewNotebook_Keyboard_Closed.pptx](https://github.com/microsoft/azuredatastudio/files/3404898/5616_.Accessibility.A11y_.Azure.Data.Studio_.NewNotebook_Keyboard_Closed.pptx)
